make/autoconf/spec.gmk.template

```diff
@@ -897,6 +898,10 @@ STATIC_LIBS_GRAAL_IMAGE_DIR := $(IMAGES_OUTPUTDIR)/$(STATIC_LIBS_GRAAL_IMAGE_SUB
 GRAAL_BUILDER_IMAGE_SUBDIR := graal-builder-jdk
 GRAAL_BUILDER_IMAGE_DIR := $(IMAGES_OUTPUTDIR)/$(GRAAL_BUILDER_IMAGE_SUBDIR)
 
+# Runtime jdk link image
+RL_INTERMEDIATE_IMAGE_SUBDIR := runtime-link-initial-jdk
+RL_INTERMEDIATE_IMAGE_DIR := $(IMAGES_OUTPUTDIR)/$(RL_INTERMEDIATE_IMAGE_SUBDIR)
+
```

Comment:
This intermediate directory is only used inside the Images.gmk. I don't think we need to define it in spec. It can be a local variable in Images.gmk. Also, unless you really want this intermediate image in "images" as some kind of deliverable I would prefer to put it somewhere under `$(SUPPORT_OUTPUTDIR)/images`.

Reply:
Fixed in the update.